src/Components/HomePage.js

```diff
@@ -134,6 +137,21 @@ const Theory = (props) => {
   );
 };
 
+const myTeam = (props) => {
+  return (
+    <View style={styles.mainContainer}>
+      <View style={styles.editorContainer}>
```

Comment:
It looks a bit "cheap" on the Team tab.
Is it a WIP layout that we should make better later, or a production-ready design?

Reply:
It's a work in progress before we add the shared drills/videos...